rgw/v2/lib/resource_op.py

```diff
@@ -355,6 +355,8 @@ def read(self, ssh_con=None):
         self.user_max_read_bytes = self.doc["config"].get("user_max_read_bytes")
         self.user_max_write_ops = self.doc["config"].get("user_max_write_ops")
         self.user_max_write_bytes = self.doc["config"].get("user_max_write_bytes")
+        self.tenanted_user = self.doc["config"].get("tenanted_user")
+        self.versioned_bucket = self.doc["config"].get("versioned_bucket")
```

Comment:
We have existing variable,
version_enable can we make use of same variable

Reply:
Updated, pls review